core-blocks/video/edit.js

```diff
@@ -86,6 +101,30 @@ class VideoEdit extends Component {
 						/>
 					</Toolbar>
 				</BlockControls>
+				<InspectorControls>
+					<PanelBody title={ __( 'Playback Controls' ) }>
```

Comment:
I know this was already changed, but I actually think "Playback controls" for audio is maybe something I should have caught before and isn't the right fit.

Calling this "playback controls" when "autoplay" and "loop" aren't "controls" per-se is weird, and it also is redundant as you have "Playback Controls > Controls".

Realistically this should just be "Video Options" and the option underneath labelled "Controls" should arguably be "Playback Controls".